### PR TITLE
add java setup to existing selenium docker recipe

### PR DESCRIPTION
**Pull request by musienko-maxim**

### CODE REVIEW

File: selenium/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,22 @@ RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/so
 
 USER user
 
+
+ENV JAVA_VERSION=8u45 \
+    JAVA_VERSION_PREFIX=1.8.0_45 \
+    
+ENV JAVA_HOME=/opt/jdk$JAVA_VERSION_PREFIX \
```
Comment (garagatyi) [inline selenium/Dockerfile]: combine these 3 ENV declarations

**Comment by garagatyi** [comment]
Can you try to improve Dockerfile by:
1. moving 20 line before 12
2. remove 18, 22 lines

File: selenium/Dockerfile [hunk 2]
```diff
@@ -21,6 +21,22 @@ RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/so
...
+
+ENV PATH=$JAVA_HOME/bin
+
+  wget \
```
Comment (garagatyi) [inline selenium/Dockerfile]: add RUN at the beggining

**Comment by benoitf** [comment]
if we need java, why not using debian_jre image ?

File: selenium/Dockerfile [hunk 3]
```diff
@@ -21,6 +21,22 @@ RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/so
...
+  --no-cookies \
+  --no-check-certificate \
+  --header "Cookie: oraclelicense=accept-securebackup-cookie" \
+  -qO- \
+  "http://download.oracle.com/otn-pub/java/jdk/$JAVA_VERSION-b14/jdk-$JAVA_VERSION-linux-x64.tar.gz" | sudo tar -zx -C /opt/ && \
```
Comment (garagatyi) [inline selenium/Dockerfile]: Remove trailing backslash

File: selenium/Dockerfile [hunk 4]
```diff
@@ -52,4 +68,4 @@ ENV DISPLAY :20.0
 CMD /usr/bin/supervisord -c /opt/supervisord.conf & \
```
Comment (garagatyi) [inline selenium/Dockerfile]: Make sure that one ampersand is OK in this case. We use 2 ampersands usually

File: selenium/Dockerfile [hunk 5]
```diff
@@ -52,4 +68,4 @@ ENV DISPLAY :20.0
...
     cd /home/user && sleep 3 && \
     ./chromedriver --port=4444 --whitelisted-ips='' & \
-    sleep 365d
\ No newline at end of file
+    sleep 365d
```
Comment (garagatyi) [inline selenium/Dockerfile]: replace `sleep 365d` with `tail -f /dev/null`
add new line at the end of the file

**Pull request closed without merging**